docs/data-tunnel/route-types/01-tss.md

```diff
@@ -1,13 +1,81 @@
 # TSS Route
 
-:::warning
+## Overview
 
-We are excited to announce that the Band v3 testnet is currently under development!
+The **TSS Route** is one of the tunnel routes used to transmit tunnel packet data from **BandChain** to other chains (currently supporting only EVM-compatible chains). This enables developers to create smart contracts that receive real-time price data from BandChain and seamlessly integrate it into their applications by utilizing both BandChain price data and Tunnel Router bridge.
 
-During this period, you may encounter intermittent updates, changes, and limited availability as we work to enhance its features and performance.
+## Creating a TSS Tunnel
 
-We appreciate your patience and understanding as we aim to deliver a robust and improved testing experience. Stay tuned for more updates and release information!
+### Deploying a Target Contract
 
-:::
+A target contract must implement the `IDataConsumer` interface to support the `BandTunnelRouter` contract. The implementation details are provided below:
 
-> This route type is currently only available on BandChain v3 testnet.
+```solidity
+interface IDataConsumer {
+    /**
+     * @dev Processes the relayed message.
+     *
+     * The relayed message must be evaluated from the tunnelRouter contract and
+     * verified by the tssVerifier contract before forwarding to the target contract.
+     *
+     * @param data The decoded tss message that is relayed from the tunnelRouter contract.
+     */
+    function process(PacketDecoder.TssMessage memory data) external;
+
+    /**
+     * @dev Activates the tunnel and set the sequence on tunnelRouter contract.
+     *
+     * This function deposits tokens into the vault and sets the latest sequence on the
+     * tunnelRouter contract if the current deposit in the vault contract exceeds a threshold.
+     * The transaction is reverted if the threshold is not met.
+     *
+     * This function should be called by the contract owner.
+     *
+     * @param latestSeq The new sequence of the tunnel.
+     */
+    function activate(uint64 latestSeq) external payable;
+
+    /**
+     * @dev Returns The tunnelRouter contract address.
+     */
+    function tunnelRouter() external view returns (address);
+
+    /**
+     * @dev Returns The tunnelId of the contract address.
+     */
+    function tunnelId() external view returns (uint64);
+}
+```
+
+### Creating a TSS Tunnel in BandChain
+
+To create a TSS Tunnel using the command-line interface (CLI), execute the following command:
+
+```bash
+bandd tx tunnel create-tunnel tss [destination-chain-id] [destination-contract-address] [encoder] [initial-deposit] [interval] [signalDeviations-json-file]
```

Comment:
should be [signal-deviations-json-file]? seems we have updated it in the code for all routes. please help update here and other routes as well.